.gitignore

```diff
@@ -0,0 +1,113 @@
+mysql-config.*
+config.json
+.sectrets
```

Comment:
Including the following two lines in .gitignore ignores all hidden files besides the .gitignore. Helps ensure all secure data usually stored in hidden files aren't committed by accident.
.*
!/.gitignore